### DISCUSSION TITLE
What are the ways to quickly run httpx cmd via Python subprocess module ?

### DISCUSSION THREAD

**Discussion by Sim4n6** [projectdiscovery/httpx#1832] [Q&A]
Hey, 

Please, I have this portion of code that slows a bit the automation:

```python
# domain length is hundreds of hundreds
for domain in domains: 
    result = subprocess.run(
        ["httpx", "-silent", "-u", domain],
        capture_output=True,
        text=True,
    )
    domain = result.stdout.strip("\n")
```

Do you have any suggestions or a way to improve it, please? any idea? 

like maybe changing the command with a specific option ? I'm just guessing for now, maybe `timeout` use or I use in a different way ? like multiprocessing approach or muti threading one ?

Regards

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.

**Reply by Sim4n6** [reply]
Never mind
